_snippets/qstash-common-request.mdx

```diff
@@ -66,3 +66,25 @@ strip efix and them to the destination API.
 
 example: "Upstash-Forward-My-Header: my-value" -> "My-Header: my-value"
 </ParamField>
+
+<ParamField header="Upstash-Callback-Forward-*" type="string" >
+ If you are using the `Upstash-Callback` header to define a callback url,
+ you can specify the headers sent along with the callback request using
+ the `Upstash-Callback-Forward-*` header.
+
+To include a header in the callback request, prefix the header name with
+`Upstash-Callback-Forward-`. We will strip efix and them to the destination API.
```

Comment:
`Upstash-Callback-Forward-`. We will strip prefix and them to the destination API.